### PR TITLE
Fix broken id

### PR DESCRIPTION
**Pull request by jguo144**
## Description
Fix error in charts_view.js caused by broken id

### CODE REVIEW

File: ckanext/c3charts/fanstatic/charts_view.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ ckan.module('c3charts_builder', function ($, _) {
                 endpoint = self.options.endpoint || self.sandbox.client.endpoint + "/api",
                 resourceView = self.options.resourceView,
                 params = setupParams(resourceView),
-                elementId = "#" + self.el.context.id,
+                elementId = "#" + self.el['0'].id,
```
Comment (surajdeval) [inline ckanext/c3charts/fanstatic/charts_view.js:30]: why did we change context to '0' in this PR?

**Review by peterVorman — APPROVED** [review]
LGTM

File: ckanext/c3charts/fanstatic/charts_view.js (hunk 1 above)
Comment (jguo144) [reply ckanext/c3charts/fanstatic/charts_view.js:30]: We just want to explicitly use the first element.

**Review by surajdeval — APPROVED** [review]
LGTM

**Pull request merged**